cmake/DockerConfig.cmake

```diff
@@ -70,93 +70,61 @@ add_custom_target(
 add_custom_target(
     centos
     COMMAND ${CMAKE_SOURCE_DIR}/docker/DockerBuild.sh
-        -u 1000
-        -g 1000
-        -v ${PROJECT_VERSION_STR}
-        -o ${MINIFI_DOCKER_OPTIONS_STR}
-        -l ${CMAKE_BINARY_DIR}
-        -d centos
-        -c BUILD_NUMBER=${BUILD_NUMBER}
-        -c DOCKER_CCACHE_DUMP_LOCATION=${DOCKER_CCACHE_DUMP_LOCATION}
-        -c DOCKER_SKIP_TESTS=${DOCKER_SKIP_TESTS}
-        -c CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-        -c DOCKER_PLATFORMS=${DOCKER_PLATFORMS}
-        -c DOCKER_PUSH=${DOCKER_PUSH}
-        -c DOCKER_TAGS=${DOCKER_TAGS}
-        -c DOCKER_BASE_IMAGE=${DOCKER_BASE_IMAGE}
+    -u 1000
+    -g 1000
+    -v ${PROJECT_VERSION_STR}
+    -o ${MINIFI_DOCKER_OPTIONS_STR}
+    -l ${CMAKE_BINARY_DIR}
+    -d centos
+    -c BUILD_NUMBER=${BUILD_NUMBER}
+    -c DOCKER_CCACHE_DUMP_LOCATION=${DOCKER_CCACHE_DUMP_LOCATION}
+    -c DOCKER_SKIP_TESTS=${DOCKER_SKIP_TESTS}
+    -c CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+    -c DOCKER_PLATFORMS=${DOCKER_PLATFORMS}
+    -c DOCKER_PUSH=${DOCKER_PUSH}
+    -c DOCKER_TAGS=${DOCKER_TAGS}
+    -c DOCKER_BASE_IMAGE=${DOCKER_BASE_IMAGE}
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/docker/)
 
 add_custom_target(
     centos-test
```

Comment:
centos-test image was only used for CI testing purposes, if it is replaced by rocky-test, then this can be removed.

Reply:
valid point 👍 https://github.com/apache/nifi-minifi-cpp/pull/1776/commits/c66f2e6fa17808f2993a4d4dd163da2797e94299